terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/ApServerName.java

```diff
@@ -0,0 +1,12 @@
+package org.terasoluna.gfw.functionaltest.app;
+
+/**
+ * Enumeration class for identifying application server.
+ * <p>
+ * If application server name is not set in application.env.properties, <br/>

```

Comment:
I think br tag need not necessary.
So I removed br tag at dbc894485791b4777f8cf25f0c2bc0d3ff96a3d6 commit.